<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlyEmProofControlForm</class>
 <widget class="QWidget" name="FlyEmProofControlForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="dvidPushButton">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Select Database</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="menuPushButton">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Menu</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="dataInfoWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="coarseBodyPushButton">
       <property name="text">
        <string>Coarse Body View</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bodyViewPushButton">
       <property name="text">
        <string>Body View</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="skeletonViewPushButton">
       <property name="text">
        <string>Skeleton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="splitPushButton">
       <property name="text">
        <string>Launch Split</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Segmentation Size:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="segmentSizeDecPushButton">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="segmentSizeIncPushButton">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fullViewPushButton">
       <property name="toolTip">
        <string>Show the segmetnation fully in the field view</string>
       </property>
       <property name="text">
        <string>Full View</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="segmentSizePushButton">
       <property name="text">
        <string>Change</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QPushButton" name="mergeSegmentPushButton">
       <property name="text">
        <string>Merge</string>
       </property>
       <property name="icon">
        <iconset resource="../gui.qrc">
         <normaloff>:/images/merge.png</normaloff>:/images/merge.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveMergePushButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="uploadPushButton">
       <property name="text">
        <string>Upload Result</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ZFlyEmBookmarkWidget" name="bookmarkWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ZFlyEmBookmarkWidget</class>
   <extends>QWidget</extends>
   <header location="global">flyem/zflyembookmarkwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../gui.qrc"/>
 </resources>
 <connections/>
</ui>
